hudi-common/src/main/java/org/apache/hudi/common/model/FileSlice.java

```diff
@@ -186,4 +186,26 @@ public boolean equals(Object o) {
   public int hashCode() {
     return Objects.hash(fileGroupId, baseInstantTime);
   }
+
+  /**
+   * Get the total file size of a file slice similar on the base file.
+   * For the log file, we need to convert its size to the estimated size similar on the base file in a certain proportion
+   */
+  public long getTotalFileSizeSimilarOnBaseFile(double logFileFraction) {
+    long logFileSize = convertLogFilesSizeSimilarOnBaseFile(logFileFraction);
+    return getBaseFile().isPresent() ? getBaseFile().get().getFileSize() + logFileSize : logFileSize;
+  }
+
+  private long convertLogFilesSizeSimilarOnBaseFile(double logFileFraction) {
```

Comment:
still think the original method name is better.

Reply:
> still think the original method name is better.

done~